test-status/web.js

```diff
@@ -142,19 +142,19 @@ exports.installWebUiRouter = (app, db) => {
     const errors = validationResult(request);
     if (!errors.isEmpty()) {
       response.render('status', Object.assign({
-        short_head_sha: request.short_head_sha,
+        shortHeadSha: request.shortHeadSha,
         is_skipping: true,
```

Comment:
Is `is_skipping` something that needs to be changed too?

Reply:
This isn't a db variable, it's a template variable, but might as well! Done.